gradle/newModule.gradle.kts

```diff
@@ -48,25 +51,29 @@ abstract class NewModule : DefaultTask() {
         val outputDir = outputDir.get()
         val file = outputDir.asFile
         val module = file.name
+        val packageName = this.packageName.get()
```

Comment:
We'd still check the package name regex, so thats already handled. Module and package names can only be the same if they match both regexes (i.e. be fully lowercased single worded), otherwise you'll get an error